assets/src/scripts/monitoring-location/selectors/hydrograph-data-selector.js

```diff
@@ -44,12 +44,21 @@ export const getIVValueRange = memoize(dataKind => createSelector(
             return null;
         }
 
-        let values = [];
-        Object.values(ivData.values).forEach(byMethodID => {
-            values.push(...byMethodID.points.filter(point => point.value !== null).map(point => point.value));
-        });
-        if (values.length) {
-            return [Math.min(...values), Math.max(...values)];
+        const result = Object.values(ivData.values).reduce((minMaxAccumulator, byMethodID) => {
```

Comment:
I had to change the implementation because the call to Math.min and Math.max fails with a Max Stack exceeded error in the error if the number of values is large (specifically for multi year time ranges).